spec/features/studios/index_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+RSpec.describe "Studio Index Page", type: :feature do
+  before :each do
+    @hollywood = Studio.create!(name: "Hollywood", location: "LA")
+    @paramount = Studio.create!(name: "Paramount Studios", location: "Culver City")
+
+    @TS = @hollywood.movies.create!(title: "Toy Story", creation_year: "2008", genre: "Fictional")
+    @Hulk = @hollywood.movies.create!(title: "Hulk", creation_year: "2010", genre: "Action")
+
+    @Dune = @paramount.movies.create!(title: "Dune", creation_year: "2021", genre: "Action")
+    @HungerGames = @paramount.movies.create!(title: "Hunger Games", creation_year: "2012", genre: "Action")
+  end
+
+  it "displays the studios name, location, and movies and corresponding attributes" do
+    visit "/studios"
+
+    within("#studio-#{@hollywood.id}") do
```

Comment:
Nice use of within blocks! You could even do movie within blocks inside this within block! But I know time is tight :)